omnisend/includes/SDK/V1/class-contact.php

```diff
@@ -189,6 +200,10 @@ public function to_array(): array {
 			$arr['lastName'] = $this->last_name;
 		}
 
+		if ( $this->phone ) {
+			$arr['phone'] = $this->phone;
```

Comment:
This is not needed. Not defined in API docs. Phone is already added (if set) in `identifiers` array.